Multisig/UI/Assets/Collectibles/CollectiblesView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CollectiblesView.swift
+//  Multisig
+//
+//  Created by Moaaz on 7/8/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct CollectiblesView: View {
+    var body: some View {
+        ZStack {
+            EmptyListPlaceholder(label: "Collectibles will appear here", image: "ico-no-collectibles")
+        }
+        .onAppear {
+            self.trackEvent(.transactions)
```

Comment:
can you please change the tracking event here?

Reply:
https://docs.google.com/spreadsheets/d/1KVaeOZ21aaNgR2jc5JM69gj0NWaOdOhlyhYXUMzh1l4/edit#gid=783259250

Reply:
Sorry, I will fix in the next pr